Remotion/ObjectBinding/Validation.UnitTests/BusinessObjectValidationResultTest.cs

```diff
@@ -46,6 +46,67 @@ public void SetUp ()
       _customerNumberInterfaceProperty = PropertyInfoAdapter.Create(MemberInfoFromExpressionUtility.GetProperty((ICustomer _) => _.CustomerNumber));
     }
 
+    [Test]
+    public void Create_WithObjectValidationFailureAndNoAffectedProperties_ReturnsSingleBusinessObjectValidationFailure ()
+    {
+      var person = new Person();
+      var objectValidationFailure = new ObjectValidationFailure(
+          person,
+          "Error",
+          "Localized error");
+      var validationResult = new ValidationResult(new[] { objectValidationFailure });
+      var businessObjectValidationResult = BusinessObjectValidationResult.Create(validationResult);
+
+      var failures = businessObjectValidationResult.GetUnhandledValidationFailures(person).ToArray();
+      Assert.That(failures.Count, Is.EqualTo(1));
+      Assert.That(failures[0].ErrorMessage, Is.EqualTo("Localized error"));
```

Comment:
Why did you skip validating the `ValidatedObject`?

Reply:
Because `UnhandledBusinessObjectValidationFailure` does not have a `ValidatedObject`